starterkits/basic/color/color.inc

```diff
@@ -15,6 +15,9 @@ $info = [
     'color-5' => t('Color 5'),
     'color-6' => t('Color 6'),
     'color-7' => t('Color 7'),
+    'color-8' => t('Color 7'),
```

Comment:
maybe we don't need to much colors in basic?

Reply:
fixed